README.md

```diff
@@ -723,6 +724,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Drishtant Dubey](https://github.com/DRISHTANT-DUBEY)
   - [Durvesh Kumar Pal](https://github.com/DurveshKumarPal)
   - [Dušan Tanasić](https://github.com/Duk4)
+  - [Chiranjeev Thomas] (https://github.com/code0monkey1)
```

Comment:
Please remove this. It is a duplicate of the other field.
```suggestion
```

Reply:
Done !